addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -100,9 +103,11 @@
 "610410","610410","Insurance","expense","False"
 "610420","610420","Travel and subsistence","expense","False"
 "610430","610430","Corporation tax expense","expense","False"
-"610440","610440","Foreign Exchange Gains/Losses","expense","False"
+"610440","610440","Foreign Exchange Losses","expense","False"
 "610450","610450","Price Differences Control","expense","False"
-"610460","610460","Cash Register Gains/Losses","expense","False"
+"610460","610460","Cash Difference Loss","expense","False"
+"610470","610470","Depreciation","expense","False"
```

Comment:
expense_depreciation?

Reply:
It's a line from the spreadsheet given to me in the task...